File: LeetCode Problems/801 to 900/867. Transpose Matrix/867. Transpose Matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>> &matrix) {
        int n = matrix.size();
        vector<vector<int>> transpos(matrix[0].size(), vector<int>(n));
        for (int i = 0; i < matrix[0].size(); i++) {
            for (int j = 0; j < n; j++) {
                transpos[i][j] = matrix[j][i];
            }
        }
        return transpos;
    }
};

File: LeetCode Problems/801 to 900/867. Transpose Matrix/867. Transpose Matrix.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        int[][] arr = new int[matrix[0].length][matrix.length];
        for (int i = 0; i < matrix[0].length; i++) {
            for (int j = 0; j < matrix.length; j++) {
                arr[i][j] = matrix[j][i];
            }
        }
        return arr;
    }
}